docs/utilities/oredictionary.md

```diff
@@ -0,0 +1,76 @@
+OreDictionary
+======
+
+The OreDictionary exists primarily for inter-mod compatibility. 
+
+By registering ores to the OreDictionary, mods make it possible to use the ores and ingots from another, similar mod in their recipes.
+
+Despite its name, the OreDictionary is used for much more than ores. Any item that is similar to another item (for example, dyes) can be registered to and used with the OreDictionary.
+
+## OreDictionary Name Convention
+
+!!! note
+    Unlike registry names, you do not want the OreDictionary name to be too unique. Follow these conventions for the most compatibility.
```

Comment:
I mention this because one time somebody asked for help, and it turned out they were using their mod id in the OreDictionary name.
I will change it to your suggestion.